index.html

```diff
@@ -279,21 +279,21 @@ <h2>Menu</h2>
             <div class="menu-ingredients">
               Plump lobster meat, mayo and crisp lettuce on a toasted bulky roll
             </div>
-          </div>
+          </div> -->
 
-        </div>
+        <!-- </div>
 
-      </div>
-    </section><!-- End Menu Section -->
+      </div> -->
```

Comment:
Why are there two "menu" sections?

Reply:
> Why are there two "menu" sections?
while we are in the edit mode there is only one menu section. 
line287:  </section> commented